runtime/vm/ContinuationHelpers.cpp

```diff
@@ -217,4 +168,37 @@ isPinnedContinuation(J9VMThread *currentThread)
 	return result;
 }
 
+UDATA
+walkContinuationStackFrames(J9VMThread *currentThread, j9object_t continuationObject, J9StackWalkState *walkState)
+{
+	Assert_VM_notNull(currentThread);
+
+	UDATA rc = J9_STACKWALK_RC_STACK_CORRUPT;
```

Comment:
Should this be initialized to success? What does it mean if the continuation is NULL? Should that also be an assert?

Reply:
if the continuation is NULL, that means the it is already terminated and cleaned up

Reply:
In that case, I think the default rc should be success - it's not an error to attempt to walk a completed continuation.